pkg/importer/analyzer/validation/nonhierarchical/management_annotation_validator.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // ValidManagementAnnotation returns an Error if the user-specified management annotation is invalid.
 func ValidManagementAnnotation(o ast.FileObject) status.Error {
-	value, found := o.GetAnnotations()[metadata.ResourceManagementKey]
-	if found && (value != metadata.ResourceManagementDisabled) {
+	value, found := o.GetAnnotations()[metadata.ManagementModeAnnotationKey]
+	if found && (value != metadata.ManagementDisabled.String()) {
```

Comment:
Should this use `metadata.IsManagementEnabled`?

Reply:
This one is special. The validation requires that IF the annotation is specified in the source, then the only valid value is `disabled`.

Reply:
Refactored to be more readable, but slightly less efficient.